polaris.shopify.com/src/components/SearchResultHighlight/SearchResultHighlight.module.scss

```diff
@@ -19,7 +19,6 @@
     [data-is-current-result='true'] &,
     [data-is-current-result]:hover & {
       box-shadow: inset 0 1px rgb(255 255 255 / 15%), 0 20px 50px rgb(0 0 0);
-      mix-blend-mode: plus-lighter;
```

Comment:
@martenbjork I'm going to ship this but meant to it to your attention yesterday. This hover effect was reducing contrast to the point where I couldn't see some lighter components in the thumbnails. There's probably a better solution here but I'll leave that with you


https://user-images.githubusercontent.com/6844391/196722560-90c1a754-3db0-4fc5-b07d-8b2a4aaa26c8.mp4 

https://user-images.githubusercontent.com/6844391/196722636-cabdbb3c-8efb-440e-a481-d437e86942b9.mp4